src/System.Data.Common/tests/System/Data/DataRowTest2.cs

```diff
@@ -2230,7 +2016,7 @@ public void Item()
 
             //  testMore();
         }
-
+        //TODO: why is this disabled?
         /*public void testMore()
```

Comment:
Why is this disabled?

Reply:
Don't know.  Feel free to delete any such commented out tests.